### PR TITLE
New html page structure

### PR DESCRIPTION
**Pull request by john4pius**
Hi,
Some changes were made to fix the highlighted issues, please help review the code.

### CODE REVIEW

**Comment by tirthajyoti-ghosh** [comment]
# Pull Request under review by T. Ghosh (TSE)

File: index.html [hunk 1]
```diff
@@ -27,15 +27,16 @@
                 </div>
                 <div class="right-bar">
                     <ul>
-                        <li><img src="src/video_post.png" alt="create video or post"></li>
-                        <li><img src="src/apps.png" alt="google applications"></li>
-                        <li><img src="src/notification.png" alt="notifications"></li>
+                        <li><img src="src/video_post.png" alt="create video or post" style="padding-bottom: 4px;"></li>
+                        <li><img src="src/apps.png" alt="google applications" ></li>
+                        <li><img src="src/notification.png" alt="notifications" style="padding-bottom: 2px;"></li>
```
Comment (tirthajyoti-ghosh) [inline index.html:32]: ### Remove inline styling. All styles should be in the `style.css` file.

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🚨

## Nice work team 🙌

Great job for using semantic tags 👏.

However, there are still some changes you need to work on.

- 👏 for bringing the `Sidebar recommended video` section beside the `main video player` section, but now the `main video player` is encroaching on the `Sidebar recommended video` section 👇
Annotation 2020-06-06 135051

- This part still has overlapping texts, as pointed out by the previous reviewer 👇
Annotation 2020-06-06 135215

- Like/Dislike buttons are still missing, as pointed out by the previous reviewer

- The share button is missing

I have left some comments below. Be sure to check them.

File: index.html (hunk 1 above)
Comment (tirthajyoti-ghosh) [inline index.html]: ### Remove inline styling. All styles should be in `style.css` file.

**Review by tirthajyoti-ghosh — COMMENTED** [review]
[OPTIONAL] Use `<nav>` tag for navigation 

*I noticed you have closed the PR from the first review (#2 ). It is not a good practice to have different PRs for different reviews. You should use the same PR for all reviews.*

Apply the changes and request for another review.
Happy coding👨‍💻

**Comment by john4pius** [comment]
> [OPTIONAL] Use `<nav>` tag for navigation
> 
> _I noticed you have closed the PR from the first review (#2 ). It is not a good practice to have different PRs for different reviews. You should use the same PR for all reviews._
> 
> Apply the changes and request for another review.
> Happy coding

I unintentionally commit a pull request on review (#2 ) so I had to revert and close it. I will work on your comment and request another review.

**Comment by elmejdki** [comment]
## Pull Request under review by _Zakariae El Mejdki_.

File: README.md [hunk 2]
```diff
@@ -1,10 +1,34 @@
...
+
+## Authors
+
+👤 **John**
+
+- Github (https://github.com/john4pius)
+
+👤 **Ahmed**
+
+* Ahmed (https://github.com/Ahmed-Benj)
```
Comment (elmejdki) [inline README.md]: **[OPTIONAL]** type your full name and also add your Linkedin and twitter profile links.

File: index.html [hunk 3]
```diff
@@ -27,179 +27,172 @@
...
-                    <iframe width="1100" height="680" src="https://www.youtube.com/embed/mENDYhfxH-o" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-                    <h2 id="video-title"><strong>Contributing to Open Source Part II: The Real Way</strong></h2>
-                </div>
-                <div>
-                    <span class="left">53.2K  &nbsp;<strong>Views</strong> <span class="subscriber-num">● March 20, 2020</span></span>
-                    <div class="right">
-                        <i id="like-button" class="fa fa-thumbs-up">44,327</i>
-                        <i id="dislike-button" class="fa fa-thumbs-down">500</i>
-                        <i id="share-button" class="fa fa-share">20k</i>
+    <!-- The flexible grid (content) -->
```
Comment (elmejdki) [inline index.html]: please remove all comments from you HTML and CSS files

File: style.css [hunk 4]
```diff
@@ -303,57 +125,63 @@ article {
...
+    padding-top: 15px;
+    font-size: 16px;
 }
 
+
+/*
+===============================================
+Account profile
+===============================================
+*/
```
Comment (elmejdki) [inline style.css]: remove all the comments from your CSS file.

File: README.md (hunk 2 above)
Comment (elmejdki) [inline README.md]: **[OPTIONAL]** put your full name there, and also add your Linkedin and twitter profile links.

File: index.html (hunk 3 above)
Comment (elmejdki) [inline index.html]: Please remove all the comments from your HTML and CSS files.
image

File: style.css [hunk 5]
```diff
@@ -1,205 +1,25 @@
...
-.container,
-.grid {
-    margin: 0 auto;
-    width: 100%;
-}
 
-.container {
-    padding: 10px;
-}
+/* Style the body */
```
Comment (elmejdki) [inline style.css]: don't forget about the comments here also, please.

File: index.html [hunk 6]
```diff
@@ -27,179 +27,172 @@
                 </div>
                 <div class="right-bar">
                     <ul>
-                        <li><img src="src/video_post.png" alt="create video or post"></li>
+                        <li><img id="post" src="src/video_post.png" alt="create video or post"></li>
```
Comment (elmejdki) [inline index.html:30]: **[OTPIONAL]** put the images inside a link tag. cause obviously they are links

File: index.html [hunk 7]
```diff
@@ -1,12 +1,12 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
```
Comment (elmejdki) [inline index.html]: please return back the lang attribute it's necessary for screen readers.

**Review by elmejdki — CHANGES_REQUESTED** [review]
# Good job so far team :tophat:, but I :thinking: **Think** you could do too much better
### status: Changes required.

I see that your website is not made for low screen resolution laptops:

I suppose that you have a big screen may be a 4k you should take on consideration that most people have a screen of 1024x768, so try to adapt your page to these screens first then go to bigger ones if you zoom in at least to 150% you will see what I'm trying to say. so next time don't forget to do that, please.

### you are almost there guys please fix the comments bellow then request a new code review.
_if you need anything contact me on slack you, you will find me as **Zakariae El Mejdki**_

**Comment by john4pius** [comment]
@elmejdki,
we will work on your recommendation.

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Excellent work on your code team 👍 
## Unfortunately, I have to tag your project as invalid! 🚫 😕 
- Your project presents the following problem when trying to merge this Pull Request:
image
- Please make proper use use of the Github Flow or Git Flow as requested [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66250306/) and [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59882084/)
- Friendly reminder: You should use Github/Git Flow in this project. Please take a step back and watch video [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59882084/). Then create a brand new repo with correct setup and feature branch (with DESCRIPTIVE name) and copy your code there or close this pull request and create another one with correct feature branch . This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready!
- Also, please write a more descriptive Pull Request description, as requested [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)
image

### Fix these issues and then request another code review!
### Comments or questions? Contact me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻 🗨️ 🧑‍💻 
## Happy Hacking! 🧠 ⌨️ 🖱️ 💻

**Pull request closed without merging**